contentcuration/contentcuration/utils/garbage_collect.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+"""
+Studio garbage collection utilities. Clean up all these old, unused records!
+"""
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.core.files.storage import default_storage as storage
+
+from contentcuration.models import ContentNode, File
+
+
+TWO_WEEKS_AGO = datetime.now() - timedelta(days=14)
+
+
+def clean_up_contentnodes(delete_older_than=TWO_WEEKS_AGO):
+    """
+    Clean up all contentnodes associated with the orphan tree with a `created`
+    time older than `delete_older_than`, as well as all files (and their file
+    objects on storage) linked to those contentnodes.
+
+    delete_older_than=The age of the contentnode from the current time, before
+    it's deleted. Default is two weeks from datetime.now().
+
+    """
+    nodes_to_clean_up = ContentNode.objects.filter(
+        parent_id=settings.ORPHANAGE_ROOT_ID,
```

Comment:
Would it be better to get all the nodes at once (in particular, thinking of ricecooker trees that might have bigger subtrees). Maybe something like 

```
garbage_node = ContentNode.objects.get(pk=settings.ORPHANAGE_ROOT_ID)
nodes_to_clean_up = ContentNode.objects.exclude(parent=None) # exclude parent node
                   .filter(tree_id=garbage_node.tree_id, modified__gt=delete_older_than) # Note: did you mean gt?